### DISCUSSION TITLE
Missing Data Points

### DISCUSSION THREAD

**Discussion by griffingdm** [weather-gov/api#354] [Q&A]
Would love to have the raw data available that makes up the detailed forecasts such as:
- cloud coverage
- chance of precipitation
- precipitation intensity
- humidity

These would be immensely helpful! As the alternative is reading the text of the forecast and the icon and creating these values...

**Comment by scadergit** [comment]
The raw data is provided by the /gridpoints endpoint without the additional /forecast path. Note, this endpoint uses duration, allowing one record to cover several hours.

If you actually mean adding additional quantitative values to the /forecast endpoint so you don't have to parse the long or short description, then that is already on the roadmap waiting to be prioritized.

**Comment by griffingdm** [comment]
Yeah I mean adding additional quantitative values to the /forecast endpoint!
Hoping for prioritization 🙏
